Sources/TSCUtility/Netrc.swift

```diff
@@ -2,11 +2,16 @@ import Foundation
 import TSCBasic
 
 /// Supplies `Authorization` header, typically to be appended to `URLRequest`
+
+// deprecated 9/2021
+@available(*, deprecated)
 public protocol AuthorizationProviding {
```

Comment:
Don't we want something like this protocol to support, e.g. authentication via keychain? Are you planning to introduce something new here?

Reply:
yes we added a new one to SwiftPM a few weeks ago, since it can also add the conformance via a protocol

Reply:
Ah sorry, I must have missed that.

Reply:
its a fairly recent change: https://github.com/apple/swift-package-manager/blob/main/Sources/Basics/AuthorizationProvider.swift

also cc @yim-lee